nbviewer/codebook/codebook_instructions.ipynb

```diff
@@ -0,0 +1,280 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "![smiley](images/logo.png)\n",
+    "\n",
+    "# Polyglot Codebook\n",
+    "---\n",
+    "* This notebook serves the need of having multiple language coding environment at same place.\n",
+    "* We will be discussing basic call using <b>native libraries</b> for few languages. The same approach can be extended to different languages with their respective <b>HTTP request</b> packages\n",
+    "* We will be using our <b>Language Detection API</b> as our toy example. The same approach can be extended to other APIs as well with their <b>respective parameters</b>.\n",
+    "* We have setup our Jupyter Notebook for the below mentioned programming languages:\n",
+    "   1. <b>Python</b>\n",
+    "   2. <b>Scala</b>\n",
+    "   3. <b>Clojure</b>\n",
+    "   4. <b>R</b>\n",
+    "   5. <b>Julia</b>\n",
+    "   6. <b>Octave</b>\n",
+    "   7. <b>Ruby</b>\n",
+    "<br>\n",
+    "* We have bunch of Text Analytics APIs hosted on our server. [Click here](http://lab.formcept.com:7777/nlp/) to know more.\n",
```

Comment:
Lets hide this URL and set to something like- localhost:<port> till we hardened the Nginx on our lab machine. Please make this change across.